.github/workflows/trigger.yml

```diff
@@ -0,0 +1,34 @@
+name: Tag & Release Binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: Version number of the release
+        default: 0.5.0
+        required: true
+
+jobs:
+  bump_version:
+    name: Version bump
+    runs-on: ubuntu-latest
```

Comment:
Does this actually bump the version number somewhere? The `release.sh` file notates all the places where a version number needs to be bumped. We should do a check somewhere that all the numbers are up-to-date, and the CI should fail if they do not match. It doesn't necessarily have to edit the version number, just force an incorrect build to fail.

Reply:
No, I can do what Kevin suggested and fetch it from somewhere in /src